hoodie-common/src/main/java/com/uber/hoodie/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -92,20 +92,29 @@ private String getPartitionPathFromFilePath(String fullPath) {
    */
   protected void init(HoodieTableMetaClient metaClient, HoodieTimeline visibleActiveTimeline) {
     this.metaClient = metaClient;
-    this.visibleActiveTimeline = visibleActiveTimeline;
+    updateTimeline(visibleActiveTimeline);
+
     // Load Pending Compaction Operations
     resetPendingCompactionOperations(
         CompactionUtils.getAllPendingCompactionOperations(metaClient).values()
             .stream().map(e -> Pair.of(e.getKey(),
             CompactionOperation.convertFromAvroRecordInstance(e.getValue()))));
   }
 
+  /**
+   * Update commits timeline
+   * @param visibleActiveTimeline  Visible Active Timeline
+   */
+  protected void updateTimeline(HoodieTimeline visibleActiveTimeline) {
```

Comment:
rename to getCommitsAndCompactionTimeline()..

Reply:
refreshTimeline()?

Reply:
Done